src/templates/en/2019/table_of_contents.html

```diff
@@ -0,0 +1,116 @@
+{% extends "en/2019/base.html" %}
+
+{% block title %}
+Table of Contents | Web Almanac 2019
+{% endblock %}
+
+{% block styles %}
+{{ super() }}
+<style>
+.main {
+  margin-bottom: 100px;
+}
+.title::before {
```

Comment:
@MSakamaki I tried an alternate approach for creating the colored overlines for titles. Take a look and let me know what you think.

![image](https://user-images.githubusercontent.com/1120896/67444533-82a2ec80-f5d7-11e9-9735-aa21f18e7cf5.png)